elvis.config

```diff
@@ -3,42 +3,41 @@
    elvis,
    [
     {config,
-     [#{dirs => ["src", "test"],
+     [#{dirs => ["src"],
         filter => "*.erl",
-        rules => [{elvis_style, line_length, #{limit => 80}},
```

Comment:
Yeah… newer versions of erlang.mk use newer versions of elvis, like gadget does.
The best thing to do will probably be to keep elvis.config as it was and eventually someone at @inaka will upgrade erlang.mk… and then we can start running make elvis again.
Sorry for the inconvenience, @zhongwencool